pkg/auth/mutual_authhandler.go

```diff
@@ -53,12 +53,15 @@ func newMutualAuthHandler(logger logrus.FieldLogger, lc hive.Lifecycle, cfg Mutu
 }
 
 type MutualAuthConfig struct {
-	MutualAuthListenerPort int `mapstructure:"mesh-auth-mutual-listener-port"`
+	MutualAuthListenerPort   int           `mapstructure:"mesh-auth-mutual-listener-port"`
+	mutualAuthConnectTimeout time.Duration `mapstructure:"mesh-auth-mutual-Connect-timeout"`
```

Comment:
typo in `mesh-auth-mutual-Connect-timeout`: need to change it to a lowercase `c`

```
2 error(s) decoding:\n\n* '' has invalid keys: mesh-auth-mutual-connect-timeout\n* '' has unset fields: mesh-auth-mutual-Connect-timeout.\nHint: field 'FooBar' matches flag 'foo-bar', or use tag `mapstructure:\"flag-name\"` to match field with flag" subsys=daemon
```